MFVNeutralino/python/NtupleCommon.py

```diff
@@ -278,6 +281,7 @@ def miniaod_ntuple_process(settings):
                          process.mfvTriggerFloats *
                          process.jmtUnpackedCandidateTracks *
                          process.mfvVertexSequence *
+                         process.vertexFilterNtuple *
```

Comment:
The change to the vertexFilter / vertexFilterNtuple looks good to me. However, as a check: have you tried to run ntuple.py with the old version of the code and the new version of the code (using the same vertexer settings, of course), to confirm that you get the same number of events that are saved (for some sample where the vertex filter is even used, i.e. likely would have to be a background sample)? This could be a useful cross-check to avoid potential bugs.

Reply:
I have kept the same default vertexing (the old one crashes with the new vertexing) while changing just a few files for this eventfilter modification and found that the vertices+events of both outputs are identical-- I tested with 100n qcdht1500: https://pekotamn.web.cern.ch/pekotamn/NtupleV27m_test_eventfilter/

Reply:
What exactly is it that crashes? What I mean is to compare:
- the old version of the code (i.e. what is in the master branch now), with the config as we've used in the past
- the new version of the code (from this branch) using the Vertexer_cfi.py settings that we expect to give identical results to the master branch

If the old settings in the new code crash, we will want to make improvements. But I'm not entirely sure which way it is right now!

Do you have this test you performed also with a signal sample such as a high mass, 10mm tbs sample? That would be a bit more statistically robust for this test, since the rate at which background passes our vertexing is so low.

Reply:
The crash was due to the jet object in a conflict loop, which was the reason we modified this part of the scripts whenever the shared-jet mitigation is on. So, I needed to turn off the part in the config file and ran the default vertexing with the new code. 

I think if all we would like to test is this modification of eventfilter chains, we can use the new code with the default vertexing setting and just compare the outputs before and after this modification. 

So, my old version = new vertexer code  + new vertexer config (default settings) + old eventfilter chains 
but your old version = old vertexer code + old vertexer config + old eventfilter chains 
and my(your) new version = new vertexer code  + new vertexer config (default settings) + new eventfilter chains

Reply:
Note that we will come back to this later. A few concerns involve (1) a few places with `event_filter_require_vertex` (such as in tracker_mapper.py) (2) the mode should change `event_filter_require_vertex`.

Reply:
Joey took a stab on this. Done.